### DISCUSSION TITLE
user-guide/group

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#86] [文档评论区]
# user-guide/group

https://zotero-chinese.com/user-guide/group.html

**Comment by xiaohaoxing** [comment]
催更催更

**Reply by northword** [reply]
催更催更 🐶

**Comment by Mazeist** [comment]
zotfile + OneDrive 好像没问题？请问为什么不建议使用zotfile的具体原因是什么呢?

**Reply by northword** [reply]
1. ZotFile 已停止维护
2. 链接的附件不支持移动端同步，移动端仅支持 官方同步和 WebDAV
3. 将附件连接到其他地方后，附件不受 Zotero 控制，更容易出现问题（如附件在资源管理器被删除，删除条目后附件不会删除，需要插件增强）
4. 见过了太多小白设置不当导致附件丢失、同步冲突等问题
5. etc...

文档里也写了，如果你确定需要这种方式且具备问题排查能力，仍可使用。

**Reply by Mazeist** [reply]
原来如此，了解学习了！感谢回复~~

**Comment by a-hahahahaha** [comment]
Zotero7的版本中，我发现无法直接将”我的文库中“某一子文件夹中的多篇文献直接拖到群组的文库中，这是为啥？

**Reply by a-hahahahaha** [reply]
目前，我试出来了两种方式：
1、在软件中，在我的文库下新建一个文件夹，然后将目标文献导入这个文件夹，再将该文件夹复制到群组的文库，这样可以保证附件同步到群组文献中，并且在本地和web中都能打开。
2、在web中可以直接将”我的文库“中某一子文件夹中的多篇文献直接拖到群组的文库中，在网页中可以通过点击附件的Original URL打开文献。（但在软件中，在群组中点击该文献时，却显示“在此路径无法找到附件：D:\****路径*****\**文献名***.pdf
可能已将其移至 Zotero 之外或删除，或者，如果文件已添加到另一台计算机上，则可能尚未同步到 zotero.org。”，这是否是因为官方存储已满导致的？）

**Reply by a-hahahahaha** [reply]
还是想知道，在软件中，能够有比上面第一种方式更为简单的操作？

**Comment by chliuhq** [comment]
能否关联我的文库和群组中的两个文件夹，在一个文件夹中修改条目另一个文件夹同步修改？
